src/FacadeAccountCreation/FacadeAccountCreation.Core/Services/ComplianceScheme/ComplianceSchemeService.cs

```diff
@@ -71,15 +72,18 @@ public async Task<HttpResponseMessage> GetComplianceSchemeForProducerAsync(Guid
     public async Task<HttpResponseMessage> GetComplianceSchemeMembersAsync(Guid userId, Guid organisationId, Guid selectedSchemeId, string? query, int pageSize, int page)
     {
         HttpResponseMessage result = null;
-        var endpointConfigValue =  $"{_config.GetSection("ComplianceSchemeEndpoints").GetSection("GetComplianceSchemeMembers").Value}";
-        var endpoint = string.Format(endpointConfigValue, organisationId, selectedSchemeId, pageSize, page, query);
+
+        var endpointConfigValue = _config.GetSection("ComplianceSchemeEndpoints").GetSection("GetComplianceSchemeMembers").Value;
+        var uriBuilder = new UriBuilder(string.Format(endpointConfigValue, organisationId, selectedSchemeId, pageSize, page, query));
+
+        string endPoint = uriBuilder.Host + uriBuilder.Path + uriBuilder.Query;
```

Comment:
Can you use something like uriBuilder.Uri instead of joining strings together?

Reply:
URI  adds http:// and that is the reason  URL  await _httpClient.GetAsync() is expecting in format    
api/connections/2e01779b-f1c4-4396-8a42-994deffee04a/person?serviceKey=packaging